lib/nolibc/include/sys/stat.h

```diff
@@ -20,23 +20,30 @@ extern "C" {
 
 #include <nolibc-internal/shareddefs.h>
 
+typedef unsigned long __kernel_ulong_t;
```

Comment:
Why `__kernel_` typedefs and not using `unsigned long` in the fields?

Reply:
@skuenzer , this was borrowed directly from the Linux structure `/usr/include/x86_64-linux-gnu/asm/stat.h`. We can update the field types. I felt it's easier to make it compatible by using that. But I guess we could also borrow the `stat.h` from Musl, that should be fully compatible with Linux API / ABI. What do you think?